HerPortal.BusinessLogic/Models/CsvFileDownloadData.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace HerPortal.BusinessLogic.Models;
+
+public class CsvFileDownloadData
+{
+    public string CustodianCode { get; set; }
+    public int Year { get; set; }
+    public int Month { get; set; }
+    public List<CsvFileDownload> Downloads { get; set; }
```

Comment:
These `CsvFileDownload` entries appear to be for all users, so the calculated properties below will be incorrect?

Reply:
Can we discuss this? I suspect it could be made a bit simpler if we just have a single DB table with
- Custodian code
- Year
- Month
- UserId
- Last downloaded date

Reply:
Updated!